### PR TITLE
Stand-alone PE support

### PR DESCRIPTION
**Pull request by gabrielesvelto**

### CODE REVIEW

File: src/windows/symbol.rs [hunk 1]
```diff
@@ -473,3 +474,26 @@ impl RvaSymbols {
...
     }
 }
+
+pub(super) fn symbolic_to_pdb_symbols(syms: PeSymbolIterator) -> PDBSymbols {
+  let mut pdb_syms = PDBSymbols::default();
+
+  for sym in syms {
+    if let Some(name) = sym.name() {
+      let pdb_symbol = PDBSymbol {
+        name: name.to_string(),
```
Comment (calixteman) [inline src/windows/symbol.rs:484]: Maybe you should try to use TypeDumper::dump_function  with an Index(0) to have a change to demangle the name and to get the parameter_size.
You can see: https://github.com/mozilla/dump_syms/blob/master/src/windows/symbol.rs#L223.

File: src/windows/pdb.rs [hunk 2]
```diff
@@ -250,24 +251,35 @@ fn get_debug_id(dbi: &DebugInformation, pi: PDBInformation) -> String {
...
+                found_unwind_info = true;
         }
-    } else if let Ok(pdb) = PdbObject::parse(&pdb_buf) {
-        cfi_writer
-            .process(&Object::Pdb(pdb))
-            .map_err(|e| e.compat())
-            .unwrap();
     }
+
+    if !found_unwind_info {
```
Comment (calixteman) [inline src/windows/pdb.rs:270]: No way to improve indentation ;) ?

**Comment by calixteman** [comment]
It'd be nice to have some green on CI.
The linting issue should be pretty easy to fix and about the checksum issue... I've no idea what's wrong here.

File: src/windows/symbol.rs (hunk 1 above)
Comment (gabrielesvelto) [reply src/windows/symbol.rs:484]: Does that apply to Symbolic symbols too? I assumed that the names would be automatically de-mangled in that case but maybe I assumed wrong.

Comment (calixteman) [reply src/windows/symbol.rs:484]: I don't know, need to check, and if parameter_size matters, then there are some heuristics to guess it.
Maybe you can try with dll we've in `test_data` dir and if they aren't insteresting maybe you can try to compile basic.cpp with the good options to have some symbols in the dll itself (see the Makefile in this dir).

**Comment by calixteman** [comment]
I re-run test task:
https://community-tc.services.mozilla.com/tasks/VYEQvhJwSzqvdWM4sInYcg/runs/1/logs/https%3A%2F%2Fcommunity-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FVYEQvhJwSzqvdWM4sInYcg%2Fruns%2F1%2Fartifacts%2Fpublic%2Flogs%2Flive.log

**Comment by calixteman** [comment]
It's likely a symbolic issue:
https://github.com/getsentry/symbolic/issues/276

**Comment by gabrielesvelto** [comment]
I think I have addressed all the review comments... but if I haven't let me know.

**Review by calixteman — APPROVED** [review]
LGTM thx

**Pull request merged**